CHANGELOG.md

```diff
@@ -66,6 +66,11 @@
 
 ### Fixes
 
+#### Plugins
```

Comment:
I believe plugin-specific changelog entries are [going away in 3.0](https://github.com/Kong/kong/pull/8772), so you will likely want to put this in the changelog at the repository root instead of here.